src/cpp/client/secure_credentials.cc

```diff
@@ -61,6 +62,7 @@ std::shared_ptr<Credentials> WrapCredentials(grpc_credentials* creds) {
 }  // namespace
 
 std::shared_ptr<Credentials> GoogleDefaultCredentials() {
+  GrpcLibrary init;  // To call grpc_init().
```

Comment:
oh I see. you're right because C callers have to already call `grpc_init` explicitly. Sorry about that. 
Now I agree with you that every entry point in the Cpp layer where this is not called should be fixed.